packages/google-cloud-access-context-manager/.repo-metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "accesscontextmanager",
+    "name_pretty": "Access Context Manager",
+    "client_documentation": "https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-access-context-manager",
+    "product_documentation": "https://cloud.google.com/access-context-manager/docs/overview",
+    "issue_tracker": "https://github.com/googleapis/google-cloud-python/issues",
+    "release_level": "preview",
+    "language": "python",
+    "library_type": "GAPIC_AUTO",
+    "repo": "googleapis/google-cloud-python",
+    "distribution_name": "google-cloud-access-context-manager",
+    "default_version": "apiVersion",
```

Comment:
`apiVersion` is a replacement string, right? At what point does that get replaced with the right version?